packages/@aws-cdk/aws-s3-deployment/test/lambda/test.py

```diff
@@ -425,7 +425,7 @@ def read_aws_out():
 #   resourceProps: map to pass to "ResourceProperties"
 #   expected_status: "SUCCESS" or "FAILED"
 def invoke_handler(requestType, resourceProps, old_resource_props=None, physical_id=None, expected_status='SUCCESS'):
-    response_url = '<response-url>'
+    response_url = 'http://<response-url>'
```

Comment:
This was needed because `Request` validates the URL passed to it, so its needs to be "valid".